atomic_reactor/cli/task.py

```diff
@@ -69,7 +69,7 @@ def binary_container_build(task_args: dict):
 
     :param task_args: CLI arguments for a binary-container-build task
     """
-    params = TaskParams.from_cli_args(task_args)
+    params = BinaryContainerBuildTaskParams.from_cli_args(task_args)
     task = BinaryBuildTask(params)
```

Comment:
I'd keep the naming consistent as `<TaskName>Params`

Reply:
+1. Should we remove `container` from `REQUIRED_PLATFORM_FOR_BINARY_CONTAINER_BUILD`, `binary-container-prebuild`, `binary-container-postbuild` etc. as well? I think the container part is a bit redundant already.

Reply:
are we completely sure? It would have to be changed in code, in pipeline/task definitions and possibly some other places to keep consistency - for example the pvc names.

Reply:
Hmm that's true, some of the names might even be scattered across different projects (like downstream playbooks). I don't mind leaving it as is

Reply:
I'll vote yes

Reply:
@hjmodi would you mind if we leave it as it is for now in this mr and change it later on?